airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_incremental.py

```diff
@@ -66,7 +66,13 @@ def configured_catalog_for_incremental_fixture(configured_catalog) -> Configured
     catalog = incremental_only_catalog(configured_catalog)
     for stream in catalog.streams:
         if not stream.cursor_field:
-            pytest.fail("Configured catalog should have cursor_field specified for all incremental streams")
+            if stream.stream.default_cursor_field:
+                stream.cursor_field = stream.stream.default_cursor_field[:]
+            else:
+                pytest.fail(
+                    "Configured catalog should have `cursor_field` " "either `default_cursor_field` specified for all incremental streams"
```

Comment:
suggested wording improvement
```suggestion
                    f"All incremental streams should either have `cursor_field` declared in the configured catalog or `default_cursor_field` specified in the catalog output by discover. Stream {stream.stream.name} does not have either property defined."
```